tests/10apidoc/03events-initial.pl

```diff
@@ -78,9 +78,10 @@ sub matrix_initialsync
 
 # A useful function which keeps track of the current eventstream token and
 #   fetches new events since it
+# $room_id may be undefined, in which case it gets events for all joined rooms.
 sub GET_new_events_for
 {
-   my ( $user ) = @_;
+   my ( $user, $room_id ) = @_;
```

Comment:
Take this as `%params` and just expand it later, so the API here doesn't have to be specific to a room ID.